shopBuddy/shopBuddy/ShoppingList+Persistent.m

```diff
@@ -0,0 +1,120 @@
+//
+//  ShoppingList+Persistent.m
+//  shopBuddy
+//
+//  Created by Rachna Gupta on 7/21/22.
+//
+
+#import "ShoppingList+Persistent.h"
+#import "Item.h"
+#import "ShoppingList.h"
+#import "Parse/Parse.h"
+#import "Item+Persistent.h"
+#import <objc/runtime.h>
+
+@implementation ShoppingList (Persistent)
+
++ (void) _createObject:(PFObject *)object {
+    [object saveInBackground];
+}
```

Comment:
You can define this as a static method, BTW it seems it does the same as the method in `Item+Persistent` so is better to move it to a separate file and define it as `static` to reuse in both places.

This method just saves to `Parse` a `PFObject` so you can reuse for other other objects as well.